<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsNewOgrConnectionBase</class>
 <widget class="QDialog" name="QgsNewOgrConnectionBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>386</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Create a New OGR Database Connection</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="grLayout1">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="GroupBox1">
     <property name="title">
      <string>Connection Information</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="8" column="0" colspan="4">
       <widget class="QPushButton" name="btnConnect">
        <property name="text">
         <string>&amp;Test Connection</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" rowspan="3" colspan="4">
       <widget class="QGroupBox" name="mAuthGroupBox">
        <property name="title">
         <string>Authentication</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>6</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>6</number>
         </property>
         <item>
          <widget class="QgsAuthSettingsWidget" name="mAuthSettingsDatabase" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>&amp;Type</string>
        </property>
        <property name="buddy">
         <cstring>cmbDatabaseTypes</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="TextLabel1_2">
        <property name="text">
         <string>&amp;Name</string>
        </property>
        <property name="buddy">
         <cstring>txtName</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="TextLabel1">
        <property name="text">
         <string>Host</string>
        </property>
        <property name="buddy">
         <cstring>txtHost</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="TextLabel2">
        <property name="text">
         <string>&amp;Database</string>
        </property>
        <property name="buddy">
         <cstring>txtDatabase</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="TextLabel2_2">
        <property name="text">
         <string>Port</string>
        </property>
        <property name="buddy">
         <cstring>txtPort</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="3">
       <widget class="QLineEdit" name="txtName">
        <property name="toolTip">
         <string>Name of the new connection</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="3">
       <widget class="QComboBox" name="cmbDatabaseTypes"/>
      </item>
      <item row="2" column="1" colspan="3">
       <widget class="QLineEdit" name="txtHost"/>
      </item>
      <item row="3" column="1" colspan="3">
       <widget class="QLineEdit" name="txtDatabase"/>
      </item>
      <item row="4" column="1" colspan="3">
       <widget class="QLineEdit" name="txtPort">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QgsAuthSettingsWidget</class>
   <extends>QWidget</extends>
   <header>qgsauthsettingswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cmbDatabaseTypes</tabstop>
  <tabstop>txtName</tabstop>
  <tabstop>txtHost</tabstop>
  <tabstop>txtDatabase</tabstop>
  <tabstop>txtPort</tabstop>
  <tabstop>btnConnect</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QgsNewOgrConnectionBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>531</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QgsNewOgrConnectionBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>155</x>
     <y>531</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
